doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
```

Comment:
There's also the "View" button in GitHub's UI

For https://github.com/ceph/ceph branches, we also auto-generate docs for every branch, eg. `http://docs.ceph.com/docs/master/architecture/` <- substitute `master` with your wip- branch's name. But you have to push to the main ceph.git, not ceph-ci.git or your own fork.